pkg/template/v2/jwk.go

```diff
@@ -34,7 +34,7 @@ func jwkPublicKeyPem(jwkjson string) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	return pemEncode(string(mpk), "PUBLIC KEY")
+	return pemEncode(mpk, "PUBLIC KEY")
```

Comment:
why do we need to change this? 🤔 it is changing the behavior of `jwkPublickKeyPem` , and I'm pretty sure if we leave it like that we can no longer use this templating function alone.

Reply:
nvm me, just looked at the rest of the changes.